grails-app/taglib/grails/plugin/sentry/SentryTagLib.groovy

```diff
@@ -0,0 +1,32 @@
+package grails.plugin.sentry
+
+import io.sentry.Sentry
+import io.sentry.SentryTraceHeader
+
+class SentryTagLib {
+
+    static final namespace = 'sentry'
+
+    SentryConfig sentryPluginConfig
+
+    def traceMeta = { attrs, body ->
```

Comment:
maybe some javadoc would be nice (in this class as in general)

Reply:
[724873e](https://github.com/agorapulse/grails-sentry/pull/56/commits/724873e8984e1c0bd8e3fa3afc779a666d24ca47)